vsintegration/src/FSharp.ProjectSystem.PropertyPages/FSharp.PropertiesPages.vbproj

```diff
@@ -34,6 +34,7 @@
     <GeneratePkgDefFile>true</GeneratePkgDefFile>
     <IncludePkgdefInVSIXContainer>true</IncludePkgdefInVSIXContainer>
     <UsePackageTargetFallbackHack>true</UsePackageTargetFallbackHack>
+    <DisableOutputPathCopying>true</DisableOutputPathCopying>
```

Comment:
We might need this on a few more projects

Reply:
At this point we shouldn't.  The only projects picking up the path copying hack from `FSharp.Directory.Build.targets` are projects converted to the SDK where the copy hack is necessary, and this property pages project.  All other projects in our repo directly write outputs to `release\net40\bin`, but because we ultimately depend on `$(TargetFramework)` from SDK projects to determine `release\net40\bin` vs. `release\coreclr\bin`, we have to copy files after the build has completed.

tl;dr - This is the only project file that can properly set `$(OutputPath)` _and_ that pulls in `FSharp.Directory.Build.targets` which defaults to the copy hack.

Reply:
OK, understood, thanks.  I thought it was about the Copy File conflicts.